snappy_wrappers/wrappers/eb_filter/wrapper.py

```diff
@@ -13,6 +13,34 @@
 else:
     cmd_fetch = "zcat {}".format(snakemake.input.vcf)
 
+step = snakemake.config["pipeline_step"]["name"]
+config = snakemake.config["step_config"][step]
+
+if "ebfilter_threshold" in config:
+    threshold = config.get("ebfilter_threshold", 0)
+elif "eb_filter" in config and "ebfilter_threshold" in config["eb_filter"]:
+    threshold = config["eb_filter"].get("ebfilter_threshold", 0)
+elif "ebfilter" in config and "ebfilter_threshold" in config["ebfilter"]:
```

Comment:
Is it intended to allow "ebfilter" as well as "eb_filter"?

Reply:
Nevermind, I believe I understood. It's "eb_filter" vs "one_ebfilter", correct?

Reply:
Yes, I think both might be used in the code. 

`eb_filter` vs `one_ebfilter`: it needed while we keep the old way of filtration (dkfz+eb_filter+oxo_g). `one_ebfilter` is the new way of filtering.